docs/docusaurus.config.js

```diff
@@ -51,6 +51,9 @@ const config = {
           },
           sidebarPath: require.resolve("./sidebars.js"),
           routeBasePath: "/",
+          editUrl: ({ docPath }) => {
+            return `https://holocron.so/github/pr/ignite/cli/main/editor/docs/docs/${docPath}`
```

Comment:
Why is it using a third party link? I'd rather point to the GitHub editor directly. On save it as well ask if you want to fork and create a new branch. This removes the reliance on a third party service.

Not doing the docs so won't block, but personally im strongly against this.

Reply:
With Holocron you have a WYSIWYG editor, easier to use for non technical people

it also has real time collaboration and you can edit users PRs code with one click instead of checking out on their fork

Reply:
Why would non technical people read the Ignite CLI docs? For building a blockchain you technical imho.

The flow now for editing a page is to grant access a third party tool to your GitHub account, while GitHub just provides an editor, and you are able to easily render it from there as well.

Reply:
Feel free to close this PR, I personally prefer writing in a WYSIWYG editor

Reply:
Ok, please let us know if you'd like to continue on this in the future, this could have been a good addition but we can not depend on an online service.